cmd/activator/main.go

```diff
@@ -271,7 +249,6 @@ func main() {
 		Throttler:     throttler,
 		GetProbeCount: getProbeCount,
 	}
-	ah = &activatorhandler.EnforceMaxContentLengthHandler{MaxContentLengthBytes: maxUploadBytes, NextHandler: ah}
 	ah = activatorhandler.NewRequestEventHandler(reqChan, ah)
 	ah = &activatorhandler.FilteringHandler{NextHandler: ah}
```

Comment:
I don't think we need the `FilteringHandler` anymore either. We needed this to handle retries that came right back to the activator.

Reply:
Cool, then I'll remove that too.